Commonalities/documentation/API-design-guidelines.md

```diff
@@ -182,6 +183,20 @@ A **number** is similar to a C or Java number, except that octal and hexadecimal
 
 Except for minor enconding details, the above-mentioned structures provide a full description of JSON language. 
 
+## 2.5 Reduce telco-specific terminology in API definitions
+CAMARA aims to produce 'intent-based' APIs, which have two key benefits:
+- for the developer: it does not assume familiarity with the network that will fulfil the API.
+- for the operator: it avoids tight-coupling, meaning the API can be fulfilled by various networks (mobile, fixed, satellite etc.)  and network versions (4/5/6G etc.)
+
+To realise these benefits it is important to reduce the use of telco-specific terminolgy/acronyms in developer-facing APIs. 
+
+CAMARA API designers should:
+- Consider and account for how the API can be fulfilled on a range of network types
+- Avoid terms/types specific to a given telco domain. If this is unavoidable, either:
```

Comment:
Could UE be added as an example of a telco specific term to avoid?

Reply:
It is difficult for people coming from the telco domain to know which telco terms are not easily understandable by outsiders :) but probably acronyms like MSISDN (vs phone_number) or AS lay in this category

Reply:
@RandyLevensalor I've added UE as an example: @jlurien good point: maybe each draft API should have a 'non telco' reviewer to catch these? Then we can make a list/suggested alternatives.